packages/edit-post/src/style.scss

```diff
@@ -58,6 +58,24 @@ body.block-editor-page {
 .components-popover,
 .components-modal__frame {
 	@include reset;
+
+	// Until checkboxes WordPress-wide are updated to match the new style,
+	// checkboxes used in metaboxes have to be slightly unstyled here.
+	// @todo: remove this entire rule once checkboxes are the same everywhere.
+	// See: https://github.com/WordPress/gutenberg/issues/18053
+	.postbox-container input[type="checkbox"],
+	.postbox-container input[type="radio"] {
+		border: $border-width solid $dark-gray-300;
+
+		&:checked {
+			background: $white;
+			border-color: $dark-gray-300;
+		}
+
+		&::before {
+			margin: -3px -4px;
+		}
+	}
```

Comment:
Would this code be better moved to the MetaBoxes stylesheet?

Reply:
Where's that? (Apologies if it's obvious)

Reply:
here https://github.com/WordPress/gutenberg/blob/master/packages/edit-post/src/components/meta-boxes/meta-boxes-area/style.scss#L2 looks like we have similar hacks